javascript/javascript1/week3/index2.js

```diff
@@ -0,0 +1,71 @@
+
+let notes = [];
+
+function saveNote(content, id) {
+  // write some code here
+  
+    let obj={
```

Comment:
obj is not a good name, I think note or you can also write like 
```
notes.push({
        content:content,
        id:id
    })
```

Reply:
you can read more about naming here
[https://github.com/chunmeishui/hyf-homework/pull/url](url)